### PR TITLE
update withsecure

### PR DESCRIPTION
**Pull request by rombernier**
context: https://github.com/SekoiaLab/platform/issues/50837

### CODE REVIEW

File: WithSecure/action_update_incident_status.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "name": "Update status on Incident",
+  "description": "Update status on Incident.",
+  "uuid": "0410768f-8d21-4aac-94d7-9ef62a8d7cfa",
+  "docker_parameters": "update_incident_status",
+  "arguments": {
+    "title": "Incident",
+    "type": "object",
+    "properties": {
+      "target": {
+        "title": "Target",
```
Comment (squioc) [inline WithSecure/action_update_incident_status.json:11]: For the 3 actions, I think that `incident_id` would be more meaningful for the customers

**Review by squioc — APPROVED** [review]
LTGM.

File: WithSecure/action_update_incident_status.json (hunk 1 above)
Comment (rombernier) [reply WithSecure/action_update_incident_status.json:11]: Yes, I change that

Comment (rombernier) [reply WithSecure/action_update_incident_status.json]: Done

**Pull request merged**